cmake/modules/RootNewMacros.cmake

```diff
@@ -895,6 +895,7 @@ endfunction()
 #---------------------------------------------------------------------------------------------------
 #---ROOT_STANDARD_LIBRARY_PACKAGE(libname
 #                                 [NO_INSTALL_HEADERS]         : don't install headers for this package
+#                                 [INLINE_HEADERS]             : include the headers in the dictionary, verbatim
```

Comment:
Why not reuse `DICTIONARY_OPTIONS` for this? (like we do with `-writeEmptyRootPCM`)

Reply:
Thanks, addressed!